src/pkg/cli/whoami.go

```diff
@@ -60,3 +66,39 @@ func Whoami(ctx context.Context, fabric client.FabricClient, provider client.Pro
 
 	return showData, nil
 }
+
```

Comment:
Any reason we are not using the [jwt.Parser](https://github.com/DefangLabs/defang/blob/main/src/pkg/auth/client.go#L303) that we are already using?